weed/shell/command_ec_common.go

```
@@ -2,133 +2,78 @@ package shell
-	dc         DataCenterId
-	rack       RackId
-	freeEcSlot int
-	// disks maps diskId -> EcDisk for disk-level balancing
-	disks map[uint32]*EcDisk
-}
-
-type CandidateEcNode struct {
-	ecNode     *EcNode
-	shardCount int
-}
+type nodeLiveness = ec.NodeLiveness
```

golangci-lint (2.12.2)

[error] type nodeLiveness is unused

(unused)